packages/oft-evm/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@layerzerolabs/oft-evm",
+  "version": "0.0.1",
+  "license": "MIT",
+  "exports": {
```

Comment:
Oh and if you could pls add the `repository` and `description` fields (even `keywords` if you're feeling adventurous)